xkcd/pkg/pbuff/convert.go

```diff
@@ -0,0 +1,60 @@
+package pbuff
+
+import "github.com/vespian/go-exercises/xkcd/pkg/types"
+
+func PBStoryFromStory(s *types.Story) *PBStory {
+	res := PBStory{
```

Comment:
Question - if this was a real project we were working on, would you agree to merge it as-is (this part to be specific) if I told you that I find this form more readable and generally try to avoid "one-liner" returns ?